pennylane_lightning/src/lightning_qubit.hpp

```diff
@@ -20,6 +20,7 @@
 #pragma once
 
 #define _USE_MATH_DEFINES
+#include <numeric>
```

Comment:
Required for `iota` when ran on Windows ([see error in log](https://github.com/PennyLaneAI/pennylane-lightning/runs/1739576949#step:7:835) and [change adding the include guard](https://github.com/PennyLaneAI/pennylane-lightning/pull/54/commits/3b08047d2b1f49b6c6ccf3acb5286689b66ea727)).